h2o-py/h2o/estimators/uplift_random_forest.py

```diff
@@ -593,6 +668,32 @@ def auuc_nbins(self):
         Number of bins to calculate Area Under Uplift Curve.
 
         Type: ``int``, defaults to ``-1``.
+
+        :examples:
+
+        >>> import h2o
+        >>> from h2o.estimators import H2OUpliftRandomForestEstimator
+        >>> h2o.init()
+        >>> data = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/uplift/criteo_uplift_13k.csv")
+        >>> predictors = ["f1", "f2", "f3", "f4", "f5", "f6","f7", "f8"]
+        >>> response = "conversion"
+        >>> data[response] = data[response].asfactor()
+        >>> treatment_column = "treatment"
+        >>> data[treatment_column] = data[treatment_column].asfactor()
+        >>> train, valid = data.split_frame(ratios=[.8], seed=1234)
+        >>> uplift_model = H2OUpliftRandomForestEstimator(ntrees=10,
+        ...                                               max_depth=5,
+        ...                                               treatment_column=treatment_column,
+        ...                                               uplift_metric="KL",
+        ...                                               min_rows=10,
+        ...                                               seed=1234,
+        ...                                               auuc_type="qini",
+        ...                                               auuc_nbins=-1)
```

Comment:
What do you think about setting the parameter to a value other than the default? 
```suggestion
        ...                                               auuc_nbins=100)
```

Reply:
Agreed. Changed the parameters to a different value across the other PRs so it would make sense to do the same here